lib/matplotlib/text.py

```diff
@@ -1862,6 +1863,7 @@ def transform(renderer) -> Transform
                                  xy,
                                  xycoords=xycoords,
                                  annotation_clip=annotation_clip)
+        self.xytext = xytext
```

Comment:
```suggestion
        self._xytext = xytext
```
If this is done it should be private.  End users should not be accessing it.

But actually I think it would be cleaner to not set this and instead use `self.get_position()` within `_check_xytext`.